### PR TITLE
Fix kwargs for fidelity

### PR DESCRIPTION
**Pull request by rmoyard**
**Context:**

This PR fixes the use of dictionnary in the tuple argument.

### CODE REVIEW

File: pennylane/qinfo/transforms.py [hunk 1]
```diff
@@ -635,6 +635,27 @@ def circuit1():
...
+            qml.RZ(y, wires=0)
+            return qml.state()
+
+        @qml.qnode(dev)
+        def circuit_ry(y, use_ry):
+            if use_ry:
+                qml.RY(y, wires=0)
+            return qml.state()
+
+    >>> fidelity(circuit_rx, circuit_ry, wires0=[0], wires1=[0])((0.1, 0.3), (0.9, {'use_ry': True})))
```
Comment (albi3ro) [inline pennylane/qinfo/transforms.py:657]:
```suggestion
    >>> fidelity(circuit_rx, circuit_ry, wires0=[0], wires1=[0])((0.1, 0.3), (0.9, {'use_ry': True}))
```

**Review by albi3ro — CHANGES_REQUESTED** [review]
unmatched `)`

**Review by albi3ro — APPROVED** [review]
Thanks for the quick fix.

File: pennylane/qinfo/transforms.py [hunk 2]
```diff
@@ -635,6 +635,27 @@ def circuit1():
     >>> jax.grad(qml.qinfo.fidelity(circuit0, circuit1, wires0=[0], wires1=[0]))((jax.numpy.array(0.3)))
     -0.14776011
 
+    There is also the possibility to pass a single dictionnary at the end of the tuple for fixing args,
```
Comment (josh146) [inline pennylane/qinfo/transforms.py:638]:
```suggestion
There is also the possibility to pass a single dictionary at the end of the tuple for fixing args,
```

Comment (josh146) [reply pennylane/qinfo/transforms.py:638]: oops, I just realized the PR is already merged 😆

**Pull request merged**